### PR TITLE
FISH-829 Create 5.2020.7 Release Notes

### PR DESCRIPTION
**Pull request by MattGill98**

### CODE REVIEW

File: docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+== Supported APIs and Applications
+
+* Jakarta EE 8
+* Java EE 8 Applications
+
+== Notes
+
+=== MicroProfile 4.0
+
+As MicroProfile moves toward the 4.0 milestone, the Payara team has simultaneously been working to ready the Payara Platform for MicroProfile 4.0 compatibility. In this Payara Platform Community Release, you can give two of the MicroProfile specification release candidates a try: MP Health 3.0 and MP JWT Auth 1.2. This technically breaks MicroProfile 3.3 support, although functionally the MicroProfile 3.3 TCK will only fail on the `@Health` annotation removal.
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc:12]: `As MicroProfile moves toward the 4.0 milestone` should be
`As MicroProfile moves toward the 4.0 final release, the Payara team ...`

**Review by rdebusscher — CHANGES_REQUESTED** [review]
Small correction ...

**Review by rdebusscher — APPROVED** [review]

File: docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* [https://github.com/payara/Payara/pull/4479[FISH-766]] - [Community -
+https://github.com/sgflt[sgflt]] Improper synchronization of session map
+* [https://github.com/payara/Payara/pull/4998[FISH-767]] - Missing
+ejb-opentracing.jar in manifest of gf-client.jar
+* [https://github.com/payara/Payara/pull/4923[FISH-777]] - [Community -
+https://github.com/avpinchuk[avpinchuk]] Exclude OpenAPI rest endpoint
+from tracing
+* [https://github.com/payara/Payara/pull/4610[FISH-779]] - [Community -
+https://github.com/borisheithecker[borisheithecker]] Honor non-standard
+realm name for client-cert login
```
Comment (cubastanley) [inline docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc:79]: Need to remove this one from the release notes pending new PR in community for Jakarta EE fix

**Review by cubastanley — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc (hunk 2 above)
Comment (OndroMih) [inline docs/modules/ROOT/pages/release-notes/release-notes-2020-7.adoc:79]:
```suggestion
```

**Pull request merged**